PostgreSQL 11 build error

Hi,

hll fails against PostgreSQL 11:

`/usr/lib64/ccache/clang -xc++ -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -fPIC -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o MurmurHash3.bc MurmurHash3.cpp
hll.c: In function 'setup_multiset':
hll.c:2760:64: error: macro "AllocSetContextCreate" passed 5 arguments, but takes just 3
                                        ALLOCSET_DEFAULT_MAXSIZE);
                                                                ^
hll.c:2756:18: error: 'AllocSetContextCreate' undeclared (first use in this function); did you mean 'SlabContextCreate'?
     tmpcontext = AllocSetContextCreate(rcontext,
                  ^~~~~~~~~~~~~~~~~~~~~
                  SlabContextCreate
hll.c:2756:18: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [<builtin>: hll.o] Error 1
make[1]: *** Waiting for unfinished jobs....
hll.c:2759:40: error: too many arguments provided to function-like macro invocation
                                       ALLOCSET_DEFAULT_INITSIZE,
                                       ^
/usr/pgsql-11/include/server/utils/memutils.h:165:9: note: macro 'AllocSetContextCreate' defined here
#define AllocSetContextCreate(parent, name, allocparams) \
        ^
hll.c:2756:18: error: use of undeclared identifier 'AllocSetContextCreate'
    tmpcontext = AllocSetContextCreate(rcontext,
                 ^
2 errors generated.
make[1]: *** [/usr/pgsql-11/lib/pgxs/src/makefiles/../../src/Makefile.global:1009: hll.bc] Error 1
make[1]: Leaving directory '/home/devrim/Documents/Devrim/Projects/repo/pgrpms/rpm/redhat/master/hll/master/postgresql-hll-2.10.2'
error: Bad exit status from /var/tmp/rpm-tmp.qcj6ua (%build)
`

Regards, Devrim

--- a/hll.c
+++ b/hll.c
@@ -2756,11 +2756,19 @@ setup_multiset(MemoryContext rcontext)
     MemoryContext oldcontext;
     multiset_t * msp;
 
+#if (PG_VERSION_NUM >= 110000)
+    tmpcontext = AllocSetContextCreateExtended(rcontext,
+                                               "multiset",
+                                               ALLOCSET_DEFAULT_MINSIZE,
+                                               ALLOCSET_DEFAULT_INITSIZE,
+                                               ALLOCSET_DEFAULT_MAXSIZE);
+#else
     tmpcontext = AllocSetContextCreate(rcontext,
                                        "multiset",
                                        ALLOCSET_DEFAULT_MINSIZE,
                                        ALLOCSET_DEFAULT_INITSIZE,
                                        ALLOCSET_DEFAULT_MAXSIZE);
+#endif
 
     oldcontext = MemoryContextSwitchTo(tmpcontext);
 

